Services/DataBaseService/Commands/CreateUserCommands.cs

```diff
@@ -0,0 +1,27 @@
+﻿using DataBaseService.Interfaces;
+using DTO;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace DataBaseService.Commands
+{
+    public class CreateUserCommand : ICommand<UserEmailPassword>
+    {
+        private readonly IRepository<UserEmailPassword> userRepository;
+
+        public CreateUserCommand([FromServices] IRepository<UserEmailPassword> userRepository)
+        {
+            this.userRepository = userRepository;
+        }
+
+        public void Execute(UserEmailPassword data)
+        {
+            //TODO Change to custom exception
+            if (string.IsNullOrEmpty(data.Email) || string.IsNullOrEmpty(data.PasswordHash))
```

Comment:
Will you replace it with fluent validation ?

Reply:
Not, we shouldn't it validates and this fale was delete from develop and I return it